devel/api.md

```diff
@@ -625,6 +627,37 @@ Type: `integer`
 
 Ethernet speed in the unit of Mbps.
 
+## `Ethernet.SRIOV_SUBTREE`
+
+Besides basic ethernet interface properties, if the ethernet device support
+Single-root input/output virtualization and is configured to use it, the
+ethernet subtree also contains a dictionary saved in key
+`Ethernet.SRIOV_SUBTREE`.
+
+### `Ethernet.SRIOV.TOTAL_VFS`
+
+Type: `integer`
+
+Total number of Virtual Functions per Physical Function.
```

Comment:
IMHO a comment like this should be added to document the dependency between the total VFs and the network state:

```
Changing the total VFs will also change the interfaces that are visible in the Nmstate state. Reducing the total VFs without setting `Interface.STATE` to `InterfaceState.ABSENT` for the corresponding interfaces might lead to an undefined system state or error.
```

What do you think?

Reply:
I think this is right. We should warn about that. Thanks for point it out :-)

Reply:
Done!